pkg/services/mssql/plan_details.go

```diff
@@ -74,6 +75,20 @@ func (d dtuPlanDetails) getProvisionSchema() service.InputParametersSchema {
 	return ips
 }
 
+func (d dtuPlanDetails) getFeProvisionSchema() service.InputParametersSchema {
```

Comment:
Same question as above. If we onboard new developers here, it might be better to use a more descriptive name for this path? I'm not sure what Fe means.

Reply:
@jeremyrickard yeah, actually I am worried about the same thing. But sadly I don't have an idea to make it elegant. I use "Fe" as a key word to indicate that "this is a 'from-existing' services related item". Replacing with "FromExisting" also reads weird to me. And in this PR, we have to use it everywhere... Appreciate your thoughts on this...

Reply:
Maybe just make it lengthy so it's clear

getProvisionSchemaForExistingIntance()

it's long, but it has clarity.